instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Comment from "./Comment";
+import CommentInput from "./CommentInput";
+let count = 0;
+class CommentSection extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      comments: props.comments,
+      comment: "",
+      count: count,
+      thisUserClicked: false
+    };
+  }
+  componentDidMount() {
+    const id = this.props.postId;
+    // alert("Got Item!");
+    if (localStorage.getItem(id)) {
+      this.setState({
+        comment: JSON.parse(localStorage.getItem(this.props.postId))
```

Comment:
this.state.comment is the one comment from your input. this.state.comments is all of your comments. So you are setting an array of objects where a string should be and then overwriting it.